core-tests/shared/src/test/scala/zio/stm/TMapSpec.scala

```diff
@@ -238,7 +238,7 @@ object TMapSpec extends ZIOBaseSpec {
         val tx =
           for {
             tmap <- TMap.make("a" -> 1, "aa" -> 2, "aaa" -> 3)
-            _    <- tmap.transformM((k, v) => STM.succeed(k.replaceAll("a", "b") -> v * 2))
+            _    <- tmap.transformSTM((k, v) => STM.succeed(k.replaceAll("a", "b") -> v * 2))
```

Comment:
❤️❤️❤️ This convention is so much nicer than before. We were using `M` to mean different things in different contexts, and that's quite apparent now.